### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by issea1015**

### CODE REVIEW

File: docs/setup-fixtures-engine.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Initializes _fixtures_ engine for a folder where Serverless service fixtures are placed (each in different top level folder)
 
-Fixtures engine provides out of a box means to extend existing fixtures (enrich/tweak configurations of existing fixutres for testing purpused) and to cleanup eventually created artifacts after testing is made
+Fixtures engine provides out of a box means to extend existing fixtures (enrich/tweak configurations of existing fixutres for testing purposed) and to cleanup eventually created artifacts after testing is made
```
Comment (medikoo) [inline docs/setup-fixtures-engine.md:5]: I belive intention was: `purposes` :)

Comment (issea1015) [reply docs/setup-fixtures-engine.md]: I might have updated without conscious haha. Updated again!

**Review by medikoo — APPROVED** [review]
Thank tyou @issea1015 !

**Comment by issea1015** [comment]
> Thank tyou @issea1015 !

typo 😛

**Pull request merged**